source/modules/MEI.stringtab.xml

```diff
@@ -104,9 +181,92 @@
     </attList>
     <remarks xml:lang="en">
       <p>The <att>startid</att> and <att>endid</att> attributes are used to indicate the <gi
-        scheme="MEI">chordMember</gi> elements on which the barre starts and finishes respectively.
-        The fret at which the barre should be created is recorded by the <att>fret</att>
+          scheme="MEI">chordMember</gi> elements on which the barre starts and finishes
+        respectively. The fret at which the barre should be created is recorded by the <att>fret</att>
         attribute.</p>
     </remarks>
   </elementSpec>
-</specGrp>
+  <elementSpec ident="course" module="MEI.stringtab" mode="add">
```

Comment:
Why are the elementSpec entries for course and tuning in both the stringtab and shared modules?

Reply:
Now deduplicated

Reply:
Also: Why `mode="add"` needed here?